pyaedt/modeler/Circuit.py

```diff
@@ -137,6 +160,20 @@ def primitives(self):
             self._primitivesDes = self._app.project_name + self._app.design_name
         return self._primitives
 
+    @property
+    def primitives(self):
+        """Primitives.
+
+        .. deprecated:: 0.4.10
+           Use :func:`Circuit.modeler.layout` instead.
```

Comment:
@maxcapodi78 Was it deprecated in 0.4.10 or will it be done in the coming 0.4.11 release?

Reply:
you are right. I'll fix it